webui/react/src/components/LogViewer/LogViewerFilters.tsx

```diff
@@ -89,15 +89,19 @@ const LogViewerFilters: React.FC<Props> = ({ onChange, onReset, options, values
             itemName={LABELS.allocationIds}
             value={values.allocationIds}
             onChange={handleChange('allocationIds', String)}>
-            {selectOptions?.allocationIds?.map(id => <Option key={id} value={id}>{id}</Option>)}
+            {selectOptions?.allocationIds?.map(id => (
+              <Option key={id} value={id}>{id || 'No Allocation'}</Option>
```

Comment:
It's not so much "no allocation" as it is "no allocation _filters_"

Reply:
@mapmeld I think this reasoning applies to every filter

Reply:
I should have asked about this before changing -
when you have a container, the options are "All Containers", "No Container", or [container-id]. Assuming that's true of the other filters, the "No _" does not mean to remove filters (as with All), but to select unassociated logs such as

```
[2022-04-26T20:17:11.219319Z] || INFO: Scheduling TensorBoard (really-casual-badger) (id: a770396c-cf53-4aae-bded-4be9eb02378f) <info> [2022-04-26 20:17:11]
[2022-04-26T20:17:11.745773Z] || INFO: TensorBoard (really-casual-badger) was assigned to an agent <info> [2022-04-26 20:17:32]
[2022-04-26T20:17:32.258493Z] || INFO: Service of TensorBoard (really-casual-badger) is available 
```

Reply:
Sure, that's fine. "No Allocation Id" meaning "logs on which there is no allocation id".

Reply:
ok I will update all 4, only one which sounds odd to me is No Rank ID (?)

Reply:
ah good point, for that one just `No Rank`?